sync3/handler/handler.go

```diff
@@ -41,13 +41,13 @@ var logger = zerolog.New(os.Stdout).With().Timestamp().Logger().Output(zerolog.C
 // This is a net.http Handler for sync v3. It is responsible for pairing requests to Conns and to
 // ensure that the sync v2 poller is running for this client.
 type SyncLiveHandler struct {
-	V2         sync2.Client
-	Storage    *state.Storage
-	V2Store    *sync2.Storage
+	V2           sync2.Client
+	Storage      *state.Storage
+	V2Store      *sync2.Storage
 	V2Sub        *pubsub.V2Sub
 	EnsurePoller *EnsurePoller
 	ConnMap      *sync3.ConnMap
-	Extensions *extensions.Handler
+	Extensions   *extensions.Handler
```

Comment:
I guess this is just GoLand formatting a file that I touched?

Reply:
Yes this is `go fmt`